### PR TITLE
Fix Mutation Issues

### PR DESCRIPTION
**Pull request by nguyenhongphat0**
Separated issue for fixing mutations

### CODE REVIEW

**Review by paul-rohrlach — APPROVED** [review]

**Comment by paul-rohrlach** [comment]
@gloriajw would you mind checking this too?  I'm fine with the changes in general, but I can't remember whether there was a particular reason we didn't use `with` scoping from the outset?

**Review by gloriajw — COMMENTED** [review]
This is a valid correction: 
with ScopedSession() as local_db_session:
But this is not:
@jwt_required()
This has to be: @jwt_required

File: user/schema.py [hunk 1]
```diff
@@ -85,10 +85,10 @@ class UpdateUser(graphene.Mutation):
     class Arguments:
         inbound = UpdateUserInput(required=True)
 
-    @jwt_required
+    @jwt_required()
```
Comment (paul-rohrlach) [inline user/schema.py:88]: Please revert this particular change

**Review by paul-rohrlach — CHANGES_REQUESTED** [review]

**Comment by nguyenhongphat0** [comment]
I got `"wrapper() got an unexpected keyword argument 'inbound'"` on updateUser mutation when these parentheses are missing, Paul 🤔 
Screenshot 2021-03-01 at 22 44 32

**Comment by paul-rohrlach** [comment]
That's interesting...

@gloriajw have you come across this before?

**Comment by gloriajw** [comment]
No, but looking now...

**Comment by nguyenhongphat0** [comment]
I don't know why but adding these parentheses make the error gone away

Screenshot 2021-03-01 at 22 52 22

**Comment by gloriajw** [comment]
But the problem is, parens invoke the decorator immediately, which is not what you want. Without parens it's a function factory, which is what you want in a decorator. 
This looks correct, I have to dig a bit and see why this isn't working:
def jwt_required(fn):
    """
    Overriding built-in decorator, to catch the exception
    thrown when signature expires.
    """
    @wraps(fn)
    def wrapper(*args, **kwargs):

**Review by paul-rohrlach — APPROVED** [review]

**Comment by gloriajw** [comment]
Phat was right! @jwt_required()

**Pull request merged**